package.json

```diff
@@ -25,61 +29,62 @@
     "node": ">=4"
   },
   "dependencies": {
-    "btoa": "^1.1.2",
-    "codemirror": "^5.13.2",
     "express": "^4.13.3",
-    "extract-text-webpack-plugin": "^1.0.1",
-    "filesize": "^3.3.0",
-    "kinto-client": "^0.8.3",
```

Comment:
Yeah, would be odd to package dependencies not actually used by the package. The npm package only exposes an executable capable of serving static assets, so we should declare the dependencies for doing so, and nothing more.

In the future if we expose an API to extend the app capabilities, that will be another story, and we'll reintroduce the actual dependencies for it. Right now, it just wouldn't make any sense.